src/pages/index.tsx

```diff
@@ -214,7 +214,7 @@ export const getServerSideProps: GetServerSideProps = async ({ locale }) => {
     )
   }
   let sortedPromotedWikis: PromotedWikisBuilder[] = []
-  if (promotedWikis?.length) {
+  if (promotedWikis) {
```

Comment:
```suggestion
  if (promotedWikis?.length) {
```
otherwise it will be true for an empty array